Persephone/Layouts/AlbumViewLayout.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AlbumViewLayout.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/2/18.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Cocoa
+
+class AlbumViewLayout: NSCollectionViewFlowLayout {
+  let maxItemWidth: CGFloat = 180
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init()
+
+    minimumLineSpacing = 20
+    minimumInteritemSpacing = 20
+    sectionInset = NSEdgeInsets(
+      top: 20,
+      left: 40,
+      bottom: 60,
+      right: 40
+    )
+  }
+
+  override func prepare() {
```

Comment:
I haven't checked the documentation, but should this be calling `super`?

Reply:
I checked and you're right.